include/strong_type/strong_type.ixx

```diff
@@ -0,0 +1,57 @@
+/*
+ * strong_type C++14/17/20 strong typedef library
+ *
+ * Copyright (C) Björn Fahller
+ *
```

Comment:
I wouldn't mind if you added your own name here. It's a substantial contribution.